webview-ui/src/kilocode/agent-manager/components/SessionDetail.tsx

```diff
@@ -344,6 +397,43 @@ function NewAgentForm() {
 							)}
 						</div>
 
+						{/* Version Count Dropdown */}
+						<div ref={versionDropdownRef} className="am-run-mode-dropdown-inline relative">
+							<StandardTooltip content={t("sessionDetail.versionsTooltip")}>
+								<button
+									className="am-run-mode-trigger-inline"
+									onClick={() => setIsVersionDropdownOpen(!isVersionDropdownOpen)}
+									disabled={isStarting}
+									type="button"
+									title={t("sessionDetail.versions")}>
+									<Layers size={14} />
+									<span className="am-version-count">{versionCount}</span>
+									<ChevronDown
+										size={10}
+										className={cn("am-chevron", isVersionDropdownOpen && "am-open")}
+									/>
+								</button>
+							</StandardTooltip>
+							{isVersionDropdownOpen && (
+								<div className="am-run-mode-menu-inline">
+									{([1, 2, 3, 4] as VersionCount[]).map((count) => (
```

Comment:
dont we want the number somewhere as a constant? It feels kind of tricky to have this '4' at multiple points at the code, especially since I don't see a reason I don't want to run 7